package.json

```diff
@@ -16,6 +16,7 @@
     "next": "13.4.8",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "router": "^1.3.8",
```

Comment:
this should be uninstalled again:

`npm uninstall router`

Reply:
Can you please clarify why, as we use "router" to fetch data from database by id? Thanks :)

Reply:
You are using this package?
https://www.npmjs.com/package/router

Can you point me to where you do that?

Reply:
We're using ```const router = useRouter()``` in our component PlantDetails.js to get the plant data for the specific id.

Reply:
Done, package is uninstalled